flycheck.el

```diff
@@ -3945,30 +3951,40 @@ show the icon."
     (error "Invalid fringe side: %S" side))
   (propertize "!" 'display
               (list side
-                    (flycheck-error-level-fringe-bitmap level)
+                    (flycheck-error-level-fringe-bitmap level
+                                                        (= (car (window-fringes)) 16))
                     (flycheck-error-level-fringe-face level))))
 
 
 ;;; Built-in error levels
 (when (fboundp 'define-fringe-bitmap)
   (define-fringe-bitmap 'flycheck-fringe-bitmap-double-arrow
-    (vector #b00000000
-            #b00000000
-            #b00000000
-            #b00000000
-            #b00000000
-            #b10011000
-            #b01101100
-            #b00110110
-            #b00011011
-            #b00110110
-            #b01101100
-            #b10011000
-            #b00000000
-            #b00000000
-            #b00000000
-            #b00000000
-            #b00000000)))
+    [#b11011000
+     #b01101100
+     #b00110110
+     #b00011011
+     #b00110110
+     #b01101100
+     #b11011000
+     #b00000000])
+  (define-fringe-bitmap 'flycheck-fringe-bitmap-double-arrow-hi-res
+    [#b0000000000000000
+     #b0000000000000000
+     #b1111001111000000
+     #b0111100111100000
+     #b0011110011110000
+     #b0001111001111000
+     #b0000111100111100
+     #b0000011110011110
+     #b0000011110011110
```

Comment:
No pointy end?

Reply:
I had tried that, but I didn't like the look.

Reply:
Ok, I tried it and I'm convinced :)